lib/workos/profile.rb

```diff
@@ -48,22 +48,20 @@ def to_json(*)
 
     private
 
-    # rubocop:disable Metrics/AbcSize
     sig { params(json_string: String).returns(WorkOS::Types::ProfileStruct) }
     def parse_json(json_string)
       hash = JSON.parse(json_string, symbolize_names: true)
 
       WorkOS::Types::ProfileStruct.new(
-        id: hash[:profile][:id],
-        email: hash[:profile][:email],
-        first_name: hash[:profile][:first_name],
-        last_name: hash[:profile][:last_name],
-        connection_id: hash[:profile][:connection_id],
-        connection_type: hash[:profile][:connection_type],
-        idp_id: hash[:profile][:idp_id],
-        raw_attributes: hash[:profile][:raw_attributes],
+        id: hash[:id],
```

Comment:
This was previously linked to the data structure that came from `ProfileAndToken`. I've modified that so the generation of the Profile struct assumes we are receiving only the profile hash. I've also modified the `ProfileAndToken` code to send only the Profile.

Reply:
🙏🏻  Thanks so much! I had originally wanted to do this when I first introduced `ProfileAndToken`.